starter-code/functions-and-arrays.js

```diff
@@ -43,9 +71,14 @@ console.log(average);
 
 // Array of Strings
 function averageWordLength (array) {
-
+  var total=0;
+  array.forEach(function(number){
```

Comment:
Good job. However, be careful with variable's names. Don't call it number if it is a word.